adapters/base_chat_completion.py

```diff
@@ -0,0 +1,92 @@
+from openai import OpenAI
+import openai
+import dtlpy as dl
+import os
+import logging
+
+logger = logging.getLogger("BaseChatCompletionModelAdapter")
+
+
+class ModelAdapter(dl.BaseModelAdapter):
+
+    def load(self, local_path, **kwargs):
+        api_key = os.environ.get("DATABRICKS_API_KEY", None)
+        if api_key is None:
+            raise ValueError("Missing API key: DATABRICKS_API_KEY")
+
+        self.client = OpenAI(
+            api_key=api_key,
+            base_url=self.model_entity.configuration.get("base_url")
+        )
+
```

Comment:
add here also the url check? like in the base embedder

Reply:
Done ✅